### PR TITLE
Key metadata

### PR DESCRIPTION
**Pull request by sqrl**
Made some modifications. Stripped down some logic. Should be persistent now.

KNOWN ISSUE: There's a bug in the structure which requires the storage direction in packrat.ini to be an absolute path.

### CODE REVIEW

File: packrat/file_cache.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+        Args:
+            key (string): The key to look up.
+
+        Returns:
+            A tuple with the filename on disk and filename to return to the client.
+        """
+        if key not in self.ordered_items:
+            return None
+        self.ordered_items.move_to_end(key)
```
Comment (kevinlaube) [inline packrat/file_cache.py:235]: Should we be passing in last=False here to move the key to the beginning?

Comment (sqrl) [inline packrat/file_cache.py:235]: Good catch. I misread LIFO for FIFO.

Comment (sqrl) [inline packrat/file_cache.py:235]: Fixed.

**Comment by kevinlaube** [comment]
Looks good to me!

**Pull request merged**